packages/assets/scss/01-atoms/_colored-heading-from-left.scss

```diff
@@ -0,0 +1,64 @@
+@use "00-base/configure" as *;
```

Comment:
why is it necessary to create a separate patternlab from colored-heading? should it be a variant of the existing pattern instead?

Reply:
<img width="1080" alt="Screen Shot 2022-05-06 at 6 12 57 PM" src="https://user-images.githubusercontent.com/5789411/167222836-30dd5531-9631-4a4b-95bf-a0d9cfa6d1d1.png">
<img width="904" alt="Screen Shot 2022-05-06 at 6 08 06 PM" src="https://user-images.githubusercontent.com/5789411/167222843-e3deb8c1-9d95-427e-a6fc-02f50a5416ae.png">
<img width="989" alt="Screen Shot 2022-05-06 at 6 07 32 PM" src="https://user-images.githubusercontent.com/5789411/167222854-fa2e4490-233c-44bc-9b96-eb1898bf9519.png">

After digging in more, it's more involved in consolidating these similar patterns into the exisiting mixin of `ma-heading-colored`, it's used as a sidebar heading style and in the table of content it's not using the colored heading component and it's setting the styles differently. I will make a separate folllowup/cleanup ticket to capture the issues around these components. Thanks!